benchmarks/dynamo/common.py

```diff
@@ -1616,6 +1616,40 @@ def _export(
         return onnx_program
 
 
+class OnnxModelFromDynamoAotOptimize(OnnxModelFromDynamo):
+    """Dynamo and Fx based export, with AOT optimize post export. `torch.onnx.dynamo_export`."""
+
+    _COMPILER_NAME = "dynamo_aot_optimize"
+
+    def _export(
+        self, model, example_inputs, output_path: str
+    ) -> torch.onnx.ONNXProgram:
+        if self.copy_before_export:
+            # Deepcopy model before export to avoid modification to baseline model.
+            model, example_inputs = self.deepcopy_model_and_inputs_to_device(
```

Comment:
maybe wrap this on a try-except in case input/model has random generators or other non deep-copiable stuff? or do we want to fail in these cases?

Reply:
Might be something for exploration in the future. I wouldn't want to change it in this PR, it is the same procedure for all compilers at the moment.